SIL.WritingSystems/IetfLanguageTag.cs

```diff
@@ -60,7 +60,14 @@ static IetfLanguageTag()
 			PrivateUsePattern = new Regex("\\A(" + PrivateUseSubExpr + ")\\z", RegexOptions.ExplicitCapture);
 		}
 
-		public static bool TryGetVariantSubtags(string variantCodes, out IEnumerable<VariantSubtag> variantSubtags)
+		/// <summary>
+		/// This method is used to create variantSubtags based on variant code
+		/// </summary>
+		/// <param name="variantCodes">Variant code</param>
+		/// <param name="variantSubtags">return as variantsubtags</param>
+		/// <param name="variantNames">Variant name is important for creating customized writing system</param>
```

Comment:
please change the comment this way:
Change the summary to 'Return Variant Subtags based on the code, looking up and returning subtags for standard codes or creating new ones for custom (private use) codes' 
Change the comment on variantNames to 'comma separated list of variant names to use with each private use code in variantCodes'